### DISCUSSION TITLE
Scrum Prep Milestone #2 - Last Week

### DISCUSSION THREAD

**Discussion by YanehRuiz** [uprm-inso4116-2024-2025-s1/semester-project-SafeRUM#314] [Weekly Retrospective]
Weekly Development Update: Fifth and last week of Milestone 2

Hello team!

Please provide a brief comment detailing what you accomplished this week, including any challenges faced or insights gained. Make sure to submit your updates by the deadline, which is before: Friday, October 25, 2024 @ 11am.

Thank you for your collaboration!

As for my collaboration:
I've graded, approved and reviewed issues. I've also worked 1:1 with some devs to fix some barriers. The TLs meetings are held differently now, for this week, but communication is still flowing within devs and TLs.

**Comment by noel-vargas** [comment]
I worked on testing the profile management tab and I'm currently working on 2 lecture topic tasks

**Reply by Rodal1** [reply]
I worked on regrouping the different tabs and a lecture topic task, currently working on a topic task.

**Comment by RaquelQuinones** [comment]
I finished task #15 , #16 and working on #237 and #184.

**Comment by CaribbeanCool** [comment]
I worked on documentation task #169 and lecture topic tasks with my team

**Comment by NathBat09** [comment]
Worked on lecture topic task #284

**Comment by evnd04** [comment]
This week has been difficult for me but ive been consistently grading, approving and closing issues, also been working with devs to try and fix some errors in the code and answering and asking questions on some issues.

**Comment by javier-rivera8** [comment]
This week, I worked on my tasks #194, #192, and the lecture topic task #259, successfully completing all of them.

**Comment by yadrielsamir** [comment]
Got to conduct weekly recompilation of team progress and finished with tasks #230 and #290

**Comment by Jjulianirizarry24** [comment]
Finished tasks for milestone #2, making plan to discuss with team for milestone #3

**Comment by johnsantiago9** [comment]
During this week I have finish my part in the creation in the sign up screen and login screen and making sure theres not any bug present, and I also worked by making a system where after a period of inactivity the users automatically logged out. During this task I've been having difficulty using the option to move through screens.

**Comment by williammartinez10** [comment]
Worked on #237 and #246

**Comment by JorgeRivera94** [comment]
Worked on documentation tasks #226 and #228 that refined existing sections, implementing Milestone 1 documentation feedback.

**Comment by lex939** [comment]
TL Approved and reviewed several issues. Putting the final finishing touches to the proposal document before this milestone ends tonight along with the rest of the hybrids.

**Comment by ghost** [comment]
Completed all my tasks this week. 😎

**Comment by JorgeC08** [comment]
During this week I finished my last 2 task for this milestone. Already looking what needs to be done for the next one.

**Comment by Alejandro-DeJesus** [comment]
During this week I worked on finishing issue #284, along side the rest of my team.

**Comment by jonVargas1** [comment]
Completed #275 and have worked on completing #284

**Comment by misaelmercado1** [comment]
Implemented button for listing locations and reports associated with them and worked on documentation tasks

**Comment by joel-alvarado** [comment]
During this week I working on the security of the application, some boilerplate for an email verification system, and connecting the current database to the Firebase Authentication system currently present.

**Comment by shenied-maldonado** [comment]
This week I focused on leading my team & completing LTTs.

**Comment by JuanJRivera** [comment]
This week, I have been working on AddLayersArchitectureDiagramToDocumentation and at the same time, I have been working on Document the Progress of PropFlow in React Native Application.
